stories/molecules/components/stepper/Stepper.stories.mdx

```diff
@@ -117,15 +131,49 @@ It is then up to the user to define the size (height and width) of the stepper,
         )
       }
     ]
-  }}
->
-  {args => {
+    const { state, dispatch } = useStepper(
+      'step1',
+      steps.map(step => ({ id: step.id, status: step.status }))
+    )
     const [submitted, setSubmitted] = useState(false)
-    const handleSubmit = () => setSubmitted(true)
     const handleOpenChange = () => setSubmitted(false)
+    const handlePrevious = useCallback(() => {
+      dispatch({
+        type: 'previousClicked'
+      })
+    }, [dispatch])
+    const handleNext = useCallback(() => {
+      dispatch({
+        type: 'stepCompleted'
+      })
+    }, [dispatch])
+    const handleSubmit = useCallback(() => {
+      setSubmitted(true)
+      dispatch({
+        type: 'stepperSubmitted'
+      })
+    }, [dispatch])
+    const handleReset = useCallback(() => {
+      setSubmitted(false)
+      dispatch({
+        type: 'stepperReset',
+        payload: {
+          initialCurrentStepId: 'step1',
+          initialStepsStatus: steps.map(step => ({ id: step.id, status: step.status }))
+        }
+      })
+    }, [dispatch, steps])
     return (
-      <div className="stepper-container-x-small">
-        <Stepper {...args} onSubmit={handleSubmit} />
+      <div className="stepper-container-small">
+        <Stepper
+          {...args}
+          currentStepId={state.currentStepId}
```

Comment:
We can't do that, because the currentStepId property has to take its value by the state so that it can be updated during navigation.
So I will prevent the control of this property.